docs/fundamentals/executor/executor-api.md

```diff
@@ -193,6 +192,17 @@ class MyExecutor(Executor):
 ```
 ````
 
+### Method return
+
+Every Executor method can return 3 types: 
+
+- If you return a `DocumentArray` object, then it will be sent over to the next Executor.
+- If you return `None` or if you don't have a `return` in your method, then the original `doc` object (potentially mutated by your function) will be sent over to the next Executor.
+- If you return a `dict` object, then it will be considered as a set of parameters and sent to the next Executor. You can leverage this feature to {ref}`send config from Executor to Executor<pass-parameters>`. The original `doc` object (potentially mutated by your function) will be sent over to the next Executor.
```

Comment:
I don't think this works quite as described, because the `dict` will be put behind `parameters['__results__'][pea_id]`, so the next executor will still use essentially the same parameters

Reply:
yes, sending config from Executor to Executor I would not consider recommended